rest-api/src/test/resources/application.properties

```diff
@@ -57,7 +57,7 @@ enceladus.rest.hadoop.auth.krb5.username=hdfs@EXAMPLE.COM
 enceladus.rest.hadoop.auth.krb5.keytab=hdfs.keytab
 
 enceladus.rest.auth.inmemory.user=user
-enceladus.rest.auth.inmemory.password=chang< )eme
+enceladus.rest.auth.inmemory.password=changeme
```

Comment:
Are you sure this was a typo? 
Could it have been a way to force the user of hermes to provide their own `application.properties` while this file serving only as an example?

Reply:
You are right. After some searching I have found answer from author about goal of this "typo". Reverted in commit 0f212739bf7dfdac1b9f24004149387a52bd66a1.